yt_dlp/extractor/crunchyroll.py

```diff
@@ -698,9 +720,49 @@ def _real_extract(self, url):
         }
 
 
-class CrunchyrollBetaIE(CrunchyrollBaseIE):
+class CrunchyrollBetaBaseIE(CrunchyrollBaseIE):
+    api_domain = None
+    bucket = None
+    params = None
+
+    def _get_params(self, lang):
+        if not CrunchyrollBetaBaseIE.api_domain:
```

Comment:
```suggestion
        if not self.params:
```

Reply:
Should this actually be `self`? If an inheriting class were to override `params` it would behave rather badly. I think it should uniformly access the variable via the class name, not mix up both kinds of reference.

Reply:
>  If an inheriting class were to override params it would behave rather badly. 

That is not an actual concern here...

I'm used to using `self` unless there is a specific reason to use the class name - ie, when setting the var. But there is no reason behind that habit afair. You can keep this as-is